jenkins-dsl/print-shas.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+#
+# Print the SHAs used in all the stage jobs.
+# If they all match then print just them, else print every job's used SHA
+#
+
+if [ $(grep ") cassandra" head.txt | sort -k4 | uniq -f3 | sed -n '$=') -eq 3 ]; then
```

Comment:
Where does `head.txt` file come from? not following... I can see how `.head` files are created but not the `head.txt` one.

Reply:
fixed.